doc/nrf/config_and_build/configuring_app/zephyr_samples_sysbuild.rst

```diff
@@ -6,42 +6,46 @@ Using Zephyr samples with sysbuild
+|               ``SB_CONFIG_NETCORE_IPC_RADIO``           | :ref:`ipc_radio`                          | ✓         | ✓        | Requires additional configuration. The following Kconfigs provide predefined configurations:                          |
+|                                                         |                                           |           |          | ``SB_CONFIG_NETCORE_IPC_RADIO_BT_HCI_IPC`` enables HCI serialization                                                  |
+|                                                         |                                           |           |          | for Bluetooth, ``SB_CONFIG_NETCORE_IPC_RADIO_BT_RPC`` enables nRF RPC serialization for Bluetooth, or                 |
+|                                                         |                                           |           |          | ``SB_CONFIG_NETCORE_IPC_RADIO_IEEE802154`` enables spinel serialization for IEEE 802.15.4.                            |
++---------------------------------------------------------+-------------------------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
+|               ``SB_CONFIG_NETCORE_NONE``                | No image                                  | -         | -        |                                                                                                                       |
++---------------------------------------------------------+-------------------------------------------+-----------+----------+-----------------------------------------------------------------------------------------------------------------------+
+
+The default for Thingy:53 is the :ref:`nrf5340_empty_net_core` sample application.
 The default for other nRF53 devices is having no image added to the build.
 
-When configuring an application, such as :zephyr_file:`zephyr/samples/bluetooth/peripheral_hr`, you must configure it with a supported network core image to ensure proper functionality.
-For basic Bluetooth samples, you can use `hci_ipc`, `multiprotocol_rpmsg`, or `ipc_radio`.
-The following instructions demonstrate how to build a sample with the `hci_ipc` network image selected:
+When configuring an application, such as :ref:`zephyr:peripheral_hr`, you must configure it with a supported network core image to ensure proper functionality.
+For basic Bluetooth samples, you can use :ref:`zephyr:bluetooth-hci-ipc-sample`, :ref:`multiprotocol-rpmsg-sample`, or :ref:`ipc_radio`.
+The following instructions demonstrate how to build a sample with the :ref:`zephyr:bluetooth-hci-ipc-sample` network image selected:
 
 .. tabs::
 
     .. group-tab:: west
 
-        west build -b <board> -- -DSB_CONFIG_NETCORE_HCI_IPC=y
+       .. code-block::
+
+          west build -b <board> -- -DSB_CONFIG_NETCORE_HCI_IPC=y
 
     .. group-tab:: cmake
```

Comment:
We never provided pure CMake commands in the NCS until sysbuild migration guide. The only other occurrence is nRF70_fw_patch_update.rst. Do we really need these commands?

Reply:
@nordicjm Can you check this?